client/components/DashboardEvent.tsx

```diff
@@ -11,11 +12,18 @@ interface IDashboardEventProps {
 
 const DashboardEvent: React.FC<IDashboardEventProps> = ({ event, loading }) => {
   return (
-    <Card style={{ marginTop: '12px' }} key={`event-${event.id}`}>
+    <Card style={{ marginTop: '12px' }}>
       <ProgressCardContent loading={loading}>
-        <Typography gutterBottom variant="h5" component="h2">
-          {event.name}
-        </Typography>
+        <Link
+          href="/dashboard/events/[id]"
```

Comment:
`[id]`?

Reply:
Take a look at [next docs](https://nextjs.org/docs/api-reference/next/link#dynamic-routes) and how the files are [named](https://nextjs.org/docs/routing/dynamic-routes) after that

Reply:
oh, neato, TIL!